jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalecen" -c "base ocnanal ocnanalecen"
+
+##############################################
+# Set variables used in the script
+##############################################
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+###############################################################
+# Run relevant script
+
+EXSCRIPT=${GDASOCNCENPY:-${HOMEgfs}/ush/exgdas_global_marine_analysis_ecen.py}
```

Comment:
this is an exscript, and should be in `scripts/` not `ush/`

Reply:
fixed